### DISCUSSION TITLE
Show option when right-clicking on the page

### DISCUSSION THREAD

**Discussion by Jandev** [mjisaak/skilling-champion-extension#4] [Q&A] [answered]
The extension only appears to be working on hyperlinks at the moment.

It would be a great addition if I can right-click any page (on a supported domain) and choose the option to create a Creators Link also.

Over here, I've just right-clicked to any portion of the page.
![context menu](https://user-images.githubusercontent.com/462356/96970380-e6763300-1513-11eb-82cd-120c526ebd8c.png)
Having the option available here also would create the link `https://docs.microsoft.com/en-us/azure/templates/?WT.mc_id=my-creators-id`

**Comment by mjisaak** [comment, marked as answer]
@Jandev
Hi Jan,

thanks for the feedback. I also thought about adding the context menu on the whole site but I preferred consistency with the existing chrome "copy link address" entry, which is also only visible when right-clicking on a link. 

So the current workaround is to find the current page e. g. on the left side and click on the link:
![image](https://user-images.githubusercontent.com/8446917/97151091-52a0a300-176f-11eb-8ccb-b4b51dfa2cbc.png)

If more users request this change, I will implement it, for now, it is as designed.

**Comment by mjisaak** [comment]
@Jandev This feature is now implemented with #24 !
